journal/original/2-journal.adoc

```diff
@@ -8132,8 +8177,7 @@ as long as life and recollection are lengthened out to me, who then did, and sti
 feel the great need there is of having the mind stayed on that which will support,
 when all outward consolation fails.`"
 
-[.small-break]
-'''
+--
 
 During the last visit made by Henry Hull to Ohio,
 the Meeting for Sufferings of that Yearly Meeting directed an
```

Comment:
Same here as with modernized, wrap an embedded "minute" around the chunk below, starting just before "Soon after this meeting directed the reprinting"